### DISCUSSION TITLE
Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by rickdroio** [EasyEngine/easyengine#1701] [Support Requests]
Hello everyone! Has anybody already installed EE in Ubuntu 22.04? Currently I'm running in 21.10 with no issues... just double checking if I can upgrade my OS. Thanks in advance.

**Comment by itsnotrin** [comment]
Yep, running it right now with no issues!

**Comment by prionkor** [comment]
Using with ubuntu 22.04.1 without any issues

**Comment by JustDr00py** [comment]
PRETTY_NAME="Ubuntu 22.04.1 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.1 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy


Running just fine on here

**Reply by GridPen** [reply]
Seamlessly ongoing with no issue in "Ubuntu 22.04.1 LTS."

**Comment by marcoandrei** [comment]
Hello, @rickdroio! 

Using EE4 in 2 VPSes with Ubuntu 22.04. No problems so far. 

I think this is a resolved issue.
